Makefile

```diff
@@ -29,7 +29,7 @@ PASS=$(shell printf "\033[32mPASS\033[0m")
 FAIL=$(shell printf "\033[31mFAIL\033[0m")
 COLORIZE=sed ''/PASS/s//$(PASS)/'' | sed ''/FAIL/s//$(FAIL)/''
 DOCKER_NAMESPACE?=$(USER)
-DOCKER_TAG?=local
+DOCKER_TAG?=latest
```

Comment:
this seems unsafe

Reply:
I would rather build latest than local, it's more convenient. 

Note that if you are building images using this Makefile the docker namespace is `$USER` so not really docker hub user. I also assume that anybody who is pushing images is cautious.

Reply:
The assumption that "anybody who is pushing images is cautious" doesn't comfort me :-), but since it goes under `$USER` namespace it should be fine.